client/my-sites/stats/annual-site-stats/index.js

```diff
@@ -0,0 +1,113 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import QuerySiteStats from 'components/data/query-site-stats';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import {
+	isRequestingSiteStatsForQuery,
+	getSiteStatsNormalizedData,
+} from 'state/stats/lists/selectors';
+import StatsModulePlaceholder from 'my-sites/stats/stats-module/placeholder';
+import ErrorPanel from 'my-sites/stats/stats-error';
+
+class AnnualSiteStats extends Component {
+	static propTypes = {
+		requesting: PropTypes.bool,
+		siteId: PropTypes.number,
+		years: PropTypes.array,
+		translate: PropTypes.func,
+		moment: PropTypes.func,
+	};
+
+	renderContent( data ) {
+		const { translate } = this.props;
+		return (
+			<div className="annual-site-stats__content">
+				<div className="annual-site-stats__stat">
+					<div className="annual-site-stats__stat-title">{ translate( 'year' ) }</div>
+					<div className="annual-site-stats__stat-figure is-large">{ data.year }</div>
+				</div>
+				<div className="annual-site-stats__stat">
+					<div className="annual-site-stats__stat-title">{ translate( 'total posts' ) }</div>
+					<div className="annual-site-stats__stat-figure is-large">{ data.total_posts }</div>
```

Comment:
All the numeric values should be passed through `this.props.numberFormat()` to localize the thousands separator and such.